### DISCUSSION TITLE
Best current commodity hardware

### DISCUSSION THREAD

**Discussion by oerdnj** [official-stockfish/Stockfish#4770] [Q&A]
Hi, I was tasked to propose a new hardware to run current Stockfish engine - the price range is $5k-$10k.

I think that the current top hardware would be something like AMD Ryzen Threadripper PRO 5995WX (64-core 128-thread), but then some older resources mention that Stockfish doesn't scale that well with more than 64+ cores (f.e. here: https://chess.stackexchange.com/questions/24338/how-does-engine-strength-scale-with-hardware).

The other option might be to assemble 4x something smaller like 4x AMD Ryzen Threadripper PRO 5955WX (16-core 32-thread) and run the cluster version - the advantage would be that it would be possible to up scale by adding more nodes.

What would be the recommended setup? Are there any good guides?

**Comment by xuxdev** [comment]
Here are some benchmarks for reference: https://ipmanchess.yolasite.com/amd--intel-chess-bench-stockfish.php
